packages/destination-actions/src/destinations/facebook-conversions-api/addToCart/index.ts

```diff
@@ -0,0 +1,98 @@
+import { ActionDefinition, IntegrationError } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+import { content_ids, content_name, content_type, contents, currency, value, action_source, event_time, event_source_url, event_id } from '../fb-capi-properties'
+import { CURRENCY_ISO_CODES, API_VERSION } from '../constants'
+import { hash_user_data, user_data_field } from '../fb-capi-user-data'
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Add to Cart',
+  description: 'Send event when a user adds a product to the shopping cart',
+  defaultSubscription: 'type = "track" and event = "Product Added"',
+  fields: {
+    action_source: { ...action_source, required: true },
+    event_time: { ...event_time, required: true },
+    user_data: user_data_field,
+    content_ids: { ...content_ids, default: { '@path': '$.properties.product_id' } },
+    content_name: content_name,
+    content_type: content_type,
+    contents: { 
+      ...contents,
+      default: [{
+        item_price: {
+          '@path': '$.properties.price'
+        },
+        id: {
+          '@path': '$.product_id'
```

Comment:
I'm not sure what the correct syntax is when we use `arrayPath` but I think this should be found within properties.

```suggestion
          '@path': '$.properties.product_id'
```

Reply:
https://github.com/segmentio/action-destinations/blob/main/packages/core/src/mapping-kit/README.md has some examples

Reply:
Thanks, Dan! Would the default be something like this for single-product Actions like Add to Cart, View Content and Search?

```
default: {
  "@arrayPath": ["$.properties", {
    "id": { "@path": "$.product_id" },
    "item_price": { "@path": "$.price" },
    "quantity": { "@path": "$.quantity" },
  }]
}
```

And this for multi-product Actions like Purchase and Checkout Initiated?

```
default: {
  "@arrayPath": ["$.properties.products", {
    "id": { "@path": "$.product_id" },
    "item_price": { "@path": "$.price" },
    "quantity": { "@path": "$.quantity" },
  }]
}
```

Reply:
fyi `@arrayPath` should only be used for array fields. if not an array:

```
default: {
  "id": { "@path": "$.product_id" },
  "item_price": { "@path": "$.price" },
  "quantity": { "@path": "$.quantity" }
}
```